google/datalab/bigquery/commands/_bigquery.py

```diff
@@ -155,6 +155,7 @@ def _create_query_subparser(parser):
   query_parser.add_argument('--datasources', help='List of external datasources to reference in the query body',
                             nargs='+')
   query_parser.add_argument('--subqueries', help='List of subqueries to reference in the query body', nargs='+')
+  query_parser.add_argument('--parameters', help='List of subqueries to reference in the query body', nargs='+')
```

Comment:
@parthea thanks for catching that

@nikhilk We also need the query body to go in the cell. One solution I see is using yaml syntax like we do in other magics, something like:
```
%%bq query --name myquery
- query: SELECT * FROM `mydataset.mytable WHERE field1 = @param1 AND field2 >= @param2
- parameters:
  - name: param1
    type: STRING
    value: :value
  - name: param2
    type: INT64
    value: 50
```
However, I feel like putting more things in the cell body is making writing magics more difficult, it's not really much easier than using the module API anymore. It also takes away from the spirit of writing SQL freely without worrying about delimiters and line breaks, but using the above syntax will push you to think about that. Do you have a better idea?

I think using a dictionary (or list for positional?) to contain parameters is a better experience than putting everything in the cell body.